src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -397,7 +393,45 @@ function MdAutocomplete ($$mdSvgRegistry) {
             ? '<li ng-if="$mdAutocompleteCtrl.notFoundVisible()"\
                          md-autocomplete-parent-scope>' + template + '</li>'
             : '';
+      }
+
+      function getContainerType(attr) {
+        var menuContainerClass = attr.mdMenuContainerClass ? ' ' + attr.mdMenuContainerClass : '';
+
+        if (isVirtualRepeatDisabled(attr)) {
+          return '\
+            <div \
+                ng-hide="$mdAutocompleteCtrl.hidden"\
+                class="md-virtual-repeat-container md-autocomplete-suggestions-container md-whiteframe-z1' + menuContainerClass + '"\
+                ng-class="{ \'md-not-found\': $mdAutocompleteCtrl.notFoundVisible() }"\
+                role="presentation">\
+              <div class="md-autocomplete-variable-height-list-scroller" role="presentation">';
```

Comment:
Can we make this just `md-autocomplete-list-scroller` since there may be other reasons to use this than just variable height?

Reply:
absolutely!